astro/src/content/docs/lifecycle/authenticate-users/oauth/tokens.mdx

```diff
@@ -213,9 +213,14 @@ The Id Token may be returned as part of an Authentication request when the `open
   <APIField name="nonce" type="String" since="1.5.0">
   </APIField>
   <APIField name="phone_number" type="String" since="1.50.0">
-    The phone number of the User if available.
+    The phone number of the User if available. (<InlineField>user.phoneNumber</InlineField> or <InlineField>user.mobilePhone</InlineField> will be used in order of precedence).
```

Comment:
I think the use of `this` in the `oauth/scopes.mdx` doc reads better, suggesting it here and other places.
```suggestion
    The phone number of the User if available. (<InlineField>user.phoneNumber</InlineField> or <InlineField>user.mobilePhone</InlineField> will be used in this order of precedence).
```

Reply:
Fixed